src/Database/Database.php

```
@@ -9343,11 +9343,17 @@ public function getCacheKeys(string $collectionId, ?string $documentId = null, a
     {
         if ($this->adapter->getSupportForHostname()) {
             $hostname = $this->adapter->getHostname();
+            $parts = parse_url($hostname);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$parts'. (undefined)

(UnusedLocalVariable)